### DISCUSSION TITLE
QLoRa LLama3.1 405b with `qlora_sharded_model_loading: true`

### DISCUSSION THREAD

**Discussion by v-dicicco** [axolotl-ai-cloud/axolotl#1931] [Q&A]
Hello,
in order to do QLoRa with the 405b while loading the model with lower cpu ram requirements, should we enable `qlora_sharded_model_loading` [in the config](https://github.com/axolotl-ai-cloud/axolotl/blob/main/examples/llama-3/qlora-fsdp-405b.yaml)? I've seen [this commit](https://github.com/axolotl-ai-cloud/axolotl/commit/3ebf22464b30390220d22a7b5fee04815cdbc0d9), but I'm not sure since I don't see this flag in the config nor in the docs. Is this feature expected to work?

Thanks!

**Comment by NanoCode012** [comment]
It's quite nested, but yes, it does exist. I haven't personally used that config though
